virttest/bootstrap.py

```diff
@@ -955,7 +956,6 @@ def bootstrap(options, interactive=False):
                 LOG.debug("Module %s loaded", module)
 
     LOG.info("")
-    LOG.info("VT-BOOTSTRAP FINISHED")
-    LOG.debug("You may take a look at the following online docs for more info:")
-    LOG.debug(" - http://avocado-vt.readthedocs.org/")
-    LOG.debug(" - http://avocado-framework.readthedocs.org/")
+    LOG.info("vt-bootstrap finished")
```

Comment:
"Please try to avoid breakages" here is quite an interesting statement, because it assumes that I don't (or didn't) try to avoid them.  Let me explain again: fixing `--vt-no-downloads` in my understanding, required fixing its semantics, thus, its behavior.

About "damage", here's my opinion: users that deploy new versions of actively developed software, and do not check or test if behavior has changed, are being naive and/or imprudent.  The "damage" is just a consequence of that.